### ISSUE TITLE
amenity kindergarten (as node) preset currently called "... ground"

### ISSUE THREAD

**Issue by sun-geo** [openstreetmap/iD#7750]
I wonder if it could (and if it is possible) make sense to modify a bit the `amenity=kindergarten` preset, so that a node object will not called as "Preschool / Kindergarten **grounds**" (in sense of a terrain)?

Example node object:
https://www.openstreetmap.org/edit?node=6639186310

So, for node object I think I would prefer a preset name just like "Preschool / Kindergarten".
What are others think about this?

**Comment by peternewman** [comment]
It's grounds as in land that surrounds a building:
https://dictionary.cambridge.org/dictionary/english/grounds

It's because you've tagged a point and used amenity I think, if you tag an area you can have a particular building:
https://github.com/openstreetmap/iD/blob/develop/data/presets/presets/building/kindergarten.json

**Comment by sun-geo** [comment]
> It's grounds as in land that surrounds a building:
> https://dictionary.cambridge.org/dictionary/english/grounds

Yes, so for an area object like:
https://www.openstreetmap.org/edit?way=282917235
the preset text "Preschool / Kindergarten grounds" matchs quite good, execpt, if you have area which defines only the building, like:
https://www.openstreetmap.org/edit?way=256908810
In the last case the surrounding is missing.
So maybe that all cases (node / area only building / area building and surroundings) will fit the preset text, just rename the preset from "Preschool / Kindergarten grounds" to "Preschool / Kindergarten"?

**Comment by maro-21** [comment]
It's because there is only one preset for kindergarten. No matter what geometry it has.

The preset for kindergarten is "Preschool / Kindergarten grounds"
and the preset for school is "School Grounds" - it's becasue most of such amenities we map as an area including school buildings, pitches and area around them. Of course it is possible to map them as a point, or some schools are only inside one building without any grounds, but we have one preset for amenity=school. Such name is to prevent people from mapping school only with building tag.

See also:
https://github.com/openstreetmap/iD/issues/6533
https://github.com/openstreetmap/iD/issues/6899
https://github.com/openstreetmap/iD/issues/5883

**Comment by sun-geo** [comment]
> It's because there is only one preset for kindergarten. No matter what geometry it has.

Exactly, so for my understand it would be for interest, that the text is more generic => will match all forms of geometries.

Thanks for linking the other issues. Seems some other people have the same wish/request, i will read the mentioned threads later in detail.
